lib/default_tracking.dart

```diff
@@ -0,0 +1,77 @@
+/// Tracking default events.
+/// 
+/// Different platforms have different default events. 
+/// Refer to platform specific docs for more details.
+/// 
+/// * [iOS](https://www.docs.developers.amplitude.com/data/sdks/ios-swift/#tracking-default-events)
+/// * [Android](https://www.docs.developers.amplitude.com/data/sdks/android-kotlin/#tracking-default-events)
+/// * [Web](https://www.docs.developers.amplitude.com/data/sdks/browser-2/#tracking-default-events)
+
+class DefaultTrackingOptions {
+  // Available on all platforms: iOS, Android, and Web
+  final bool sessions;
+  // Available on iOS and Android
+  final bool appLifecycles;
+  final bool screenViews;
+  // Available on Android
+  final bool deepLins;
+  // Available on Web
+  final bool attribution;
+  final bool pageViews;
+  final bool formInteractions;
+  final bool fileDownloads;
```

Comment:
Similar thoughts here for platform specific options. Maybe good to nest under a Web specific config option for clarity?

Reply:
Updated in https://github.com/amplitude/Amplitude-Flutter/pull/162/commits/a96d46ad15e708f2fb4d96b1502728e2c7249fb7